src/xcode/ENA/ENA/Source/Models/Metadata/KeySubmissionMetadataService.swift

```diff
@@ -0,0 +1,88 @@
+////
```

Comment:
as I seen, you create a bunch of instances of this service, every time you need the service. Perhaps it would be better to create it only once and them inject it to every class, that needs it?

Reply:
Refactored